### PR TITLE
Add concepts section to docs.

### PR DESCRIPTION
**Pull request by jaheba**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: docs/concepts.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+```py
+predictor = estimator.train(train_data)
+forecasts = predictor.predict(data)
+```
+
+The reason to split `Estimator` and `Predictor` into two classes is that many
+models require a dedicated training step to generate a global model. This
+global model is only trained once, but is used to make predictions for all
+time-serires.
```
Comment (lostella) [inline docs/concepts.md]:
```suggestion
time-series.
```

File: docs/concepts.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+This is in contrast to local models, which are fitted on individual time-series
+and therefore try to capture the characteristics of each time-series but not
+the dataset in its entirety.
+
+Training a global model usually takes a lot of time: often hours, but sometimes
+even days. Thus, it is not feasable to train the model "offline" (on demand).
+In contrast, a local model is usually fitted "oline" as part of the request.
```
Comment (lostella) [inline docs/concepts.md]:
```suggestion
Training a global model can take a lot of time: up to hours, but sometimes
even days. Thus, it is not feasible to train the model "offline" (on demand).
In contrast, a local model is usually fitted "online" as part of the prediction.
```

However, I don't understand what this sentence means:

```
Thus, it is not feasible to train the model "offline" (on demand)
```

File: docs/concepts.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+In GluonTS models that are local are available as predictors, whilst global
```
Comment (lostella) [inline docs/concepts.md]:
```suggestion
In GluonTS, local models are directly available as predictors, whilst global
```

File: docs/concepts.md (hunk 2 above)
Comment (jaheba) [reply docs/concepts.md]: The second part of the sentence is missing. It's not feasible to do it for each prediction but needs to do be done separately.

Comment (lostella) [reply docs/concepts.md]: Do you want to add that?

Comment (jaheba) [reply docs/concepts.md]: Yes

File: docs/concepts.md [hunk 4]
```diff
@@ -51,7 +53,8 @@ In GluonTS a `Dataset` is a collection of time-series objects. Each of these
 objects has columns (or fields) which represent attributes of the
 time-series.
 
-In most models the `target`-field is the column that we want to predict:
+Most odels use the `target`-column to indicate the time-series that we want to
```
Comment (lostella) [inline docs/concepts.md]:
```suggestion
Most models use the `target`-column to indicate the time-series that we want to
```

**Review by lostella — CHANGES_REQUESTED** [review]
Just a typo, then it can go! 🚀

**Review by lostella — APPROVED** [review]

**Pull request merged**